packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -829,8 +832,12 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     }
 
     // Delay updating expanding/dismissing the submenu
-    // and only set focus if we have not already done so
+    // and only set focus if we have not already done so.
+    // Also if we are on the primary portion of a splitButton
+    // (and we know from above that we are not the expanded item)
+    // collapse the submenu
     if (hasSubmenu(item)) {
+      ev.stopPropagation();
```

Comment:
Why does this need to be here now?

Reply:
I bumped the hover handling on to the whole container of the spin button since that's what should be getting focus. This was added to make sure when we are hovering over the expandable portion of the splitButton that the event will not bubble up to the parent container. All other cases it doesn't change anything. This change now also fixes focus not updating correctly when hovering over a splitButton (in a menu), now splitButtons update focus just like the other items